bonus_guides/G_static_assets.md

```diff
@@ -1 +1,289 @@
+
+Adding support for other languages like SASS or React's JSX is as simple as this. Some plugins can be configured in `brunch-config.js`, but they will all work out of the box once installed.
+
+#### Other Things Possible With Brunch
+
+There are many more nice tricks we can do with Brunch which are not covered in this guide. Here are just a few:
+
+- It is possible to have [multiple build targets](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter04-starting-from-scratch.md#split-targets), for example, `app.js` for our code and `vendor.js` for third-party libraries
+- It is possible to control the order of concatenation of files. This might be necessary when working with JS files in `vendor` if they depend on each other.
+- Instead of manually copying third-party libraries into `web/static/vendor` we can [use Bower to download and install them](https://github.com/brunch/brunch-guide/blob/master/content/en/chapter05-using-third-party-registries.md).
+
+
+Should we want one of these, please read [the Brunch documentation](https://github.com/brunch/brunch/tree/master/docs).
+
+#### Phoenix Without Brunch
+
+Should we decide not to use Brunch in our new Phoenix project, run the project generator task with option `--no-brunch`:
+
+```
+mix phoenix.new --no-brunch my_project
+```
+
+####  Using Another Asset Management System in Phoenix
+
+To integrate another asset management system with Phoenix we will need to
+
+- configure that asset management system to put built assets into `priv/static/`.
+- let Phoenix know how to run a command which would watch asset source files and build assets on each change
+
+While the first point is clear, how does Phoenix know which command to run to launch an asset build tool in watch mode? If we open `config/dev.exs` we will find configuration key `:watchers`. `:watchers` is a list of tuples containing an executable and its arguments. That is, with a config like the following:
+
+```elixir
+  watchers: [node: ["node_modules/brunch/bin/brunch", "watch"]]
+```
+
+the command launched in the development mode is
+
+```
+node node_modules/brunch/bin/brunch  watch
+```
+
+Let's see how we can integrate [Webpack](http://webpack.github.io) to work with Phoenix. First we generate a project without Brunch support:
+
+
+```
+mix phoenix.new --no-brunch my_project
+```
+
+or remove Brunch from the project:
+
+```
+rm brunch-config.js && rm -rf node_modules/*
+```
+
+Modify `package.json`:
+
+```json
+{
+  "devDependencies": {
+    "webpack": "^1.11.0"
+  }
+}
+```
+
+Install webpack:
+
+
+```
+npm install
+```
+
+Create webpack configuration file `webpack.config.js`:
+
+```json
+module.exports = {
+  entry: "./web/static/js/app.js",
+  output: {
+    path: "./priv/static/js",
+    filename: "app.js"
+  }
+};
+```
+
+Now let's let Phoenix know how to run Webpack. Replace the `:watchers` definition by the following:
```

Comment:
"Now let's let" sounds fun to me. I wonder if it could be "Now let's configure Phoenix to run Webpack"?